packages/protocol/contracts/shared/tokenvault/ERC20Vault.sol

```diff
@@ -251,8 +260,12 @@ contract ERC20Vault is BaseVault {
         if (btokenDenylist[_op.token]) revert VAULT_BTOKEN_BLACKLISTED();
         if (msg.value < _op.fee) revert VAULT_INSUFFICIENT_FEE();
 
-        (bytes memory data, CanonicalERC20 memory ctoken, uint256 balanceChange) =
-            _handleMessage(_op);
+        (
```

Comment:
maybe we can resolve bridge before calling _handleMessage to avoid resolving it again inside sendToken

Reply:
Correct.